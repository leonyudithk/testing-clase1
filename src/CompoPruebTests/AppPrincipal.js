import React from 'react';

const AppPrincipal = ({saludo, subtitulo}) => {
    return (
        <div>
        <h1>{saludo}</h1>
        <h3>{subtitulo}</h3>
        <p>La informacion es correcta</p>
            
        </div>
    );
};

export default AppPrincipal;
