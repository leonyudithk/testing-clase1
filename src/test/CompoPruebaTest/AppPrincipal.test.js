import '@testing-library/jest-dom';
import React from 'react';
import { shallow} from "enzyme"
import AppPrincipal from "../../CompoPruebTests/AppPrincipal"

describe('Prueba al componente AppPrincipal',()=>{
    test('validar que renderiza AppPrincipal', ()=>{
        //Arreglar- preparar la mesa trabajo 
         const saludoPrueba = 'Hola Frontend-9'
         const subtituloPrueba = 'Hoy es su ultimo dia de clases pero el primero de carrera profesional'

         const wrapper = shallow(<AppPrincipal saludo={saludoPrueba} subtitulo={subtituloPrueba}/>) 

        // actuar hacer prueba al componte
        expect(wrapper).toMatchSnapshot()

    })

})