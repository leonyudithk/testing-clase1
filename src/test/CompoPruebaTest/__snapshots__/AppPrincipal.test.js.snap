// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba al componente AppPrincipal validar que renderiza AppPrincipal 1`] = `
<div>
  <h1>
    Hola Frontend-9
  </h1>
  <h3>
    Hoy es su ultimo dia de clases pero el primero de carrera profesional
  </h3>
  <p>
    La informacion es correcta
  </p>
</div>
`;
